db/config.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+require('dotenv').config();
+
+let development;
+
+switch (process.env.DEV_DB) {
+  case 'mysql':
+    development = {
+      username: process.env.DB_USERNAME,
+      password: process.env.DB_PASSWORD,
+      database: process.env.DB_NAME,
+      host: '127.0.0.1',
+      port: process.env.DB_PORT || 3306,
+      dialect: 'mysql',
+      dialectOptions: {
+        bigNumberStrings: true,
+      },
+    };
+    break;
+  default:
+    development = {
+      storage: process.env.DB_FILE || './db.sqlite3',
+      dialect: 'sqlite',
+    };
+    break;
+}
```

Comment:
Since we are not using the sequelize ORM on the project, this is pointless, right?

Reply:
Yes it is, simplified it now.